#robot's geometric parameters:

#Diameter of std. mecanum wheel
wheelDiameter: 0.1
#Length of the mobile robot from wheel center to wheel center
robotLength: 0.25
#Width from wheel center to wheel center
robotWidth: 0.27
#stddev vel x
devX: 0.1
#stddev vel y
devY: 0.1
#stddev vel z
devZ: 0.1
#stddev vel roll
devRoll: 0.1
#stddev vel pitch
devPitch: 0.1
#stddev vel x yaw
devYaw: 0.1